src/sentry/lang/native/applecrashreport.py

```diff
@@ -134,17 +134,17 @@ def _convert_frame_to_apple_string(self, frame, next=None, number=0):
             file = ""
             if frame.get("filename") and frame.get("lineno"):
                 file = " (%s:%s)" % (
-                    posixpath.basename(frame.get("filename") or NATIVE_UNKNOWN_STRING),
+                    posixpath.basename(frame.get("filename", NATIVE_UNKNOWN_STRING)),
                     frame["lineno"],
                 )
-            symbol = "%s%s" % (frame.get("function") or NATIVE_UNKNOWN_STRING, file)
+            symbol = "%s%s" % (frame.get("function", NATIVE_UNKNOWN_STRING), file)
```

Comment:
Could this key be a falsey value ever? You might still need the `or` case to handle that scenario.

Reply:
All these could be `None` explicitly, so the original implementation was safer.